dpc/src/program/noop_program.rs

```diff
@@ -26,40 +26,40 @@ use crate::{
 };
 use snarkvm_algorithms::prelude::*;
 use snarkvm_parameters::{
-    testnet1::{NoopProgramSNARKPKParameters, NoopProgramSNARKVKParameters},
+    testnet2::{NoopProgramSNARKPKParameters, NoopProgramSNARKVKParameters},
```

Comment:
Note: just to confirm that now DPC Noop Program for both testnet1 and testnet2 uses the testnet2's noop program parameters

Reply:
Good catch, will add parameterization for this.

Reply:
I will work on this. I think this can be solved by just including a name for the testnet (e.g., "testnet1") in the DPC Parametes.

Reply:
Solved in #301.